Mathlib/Analysis/InnerProductSpace/JointEigenspace.lean

```diff
@@ -135,7 +141,7 @@ theorem iSup_eigenspace_restrict [FiniteDimensional 𝕜 E] {F : Submodule 𝕜
 
 /-- The orthocomplement of the indexed supremum of joint eigenspaces of a finite commuting tuple of
 symmetric operators is trivial. -/
-theorem orthogonalComplement_iSup_iInf_eigenspaces_eq_bot [Fintype n] [FiniteDimensional 𝕜 E]
+theorem orthogonalComplement_iSup_iInf_eigenspaces_eq_bot [Fintype n]
     (T : n → (E →ₗ[𝕜] E)) (hT :(∀ (i : n), ((T i).IsSymmetric)))
     (hC : (∀ (i j : n), (T i) ∘ₗ (T j) = (T j) ∘ₗ (T i))) :
     (⨆ (γ : n → 𝕜), (⨅ (j : n), (eigenspace (T j) (γ j)) : Submodule 𝕜 E))ᗮ = ⊥ := by
```

Comment:
I'm not sure why you have such a love of parentheses. I think all these can be removed. The only ones I'm not sure about are around the `T i` and `T j` near `∘ₗ`.
```suggestion
    (T : n → E →ₗ[𝕜] E) (hT : ∀ i, (T i).IsSymmetric)
    (hC : ∀ i j, T i ∘ₗ T j = T j ∘ₗ T i) :
    (⨆ γ : n → 𝕜, ⨅ j, eigenspace (T j) (γ j) : Submodule 𝕜 E)ᗮ = ⊥ := by
```

Reply:
LOL! On paper, parentheses keep me from getting burned!

Reply:
In Lean, I'm going to have to learn how to remove them because they are very unpleasant to read...this finally gives me an appreciation for PEMDAS...I never liked that as a kid.